sycl/unittests/program_manager/arg_mask/EliminatedArgMask.cpp

```diff
@@ -304,7 +304,8 @@ TEST(EliminatedArgMask, ReuseOfHandleValues) {
     sycl::queue Queue{Dev};
     auto Ctx = Queue.get_context();
     ProgBefore = PM.getBuiltURProgram(*sycl::detail::getSyclObjImpl(Ctx),
-                                      *sycl::detail::getSyclObjImpl(Dev), Name);
+                                      *sycl::detail::getSyclObjImpl(Dev), Name)
+                     .release();
```

Comment:
Don't know how the test should be changed to work without calling `release`. Looks like it always had a leak here, and simply changing `ProgBefore/After` to `Managed<...>` breaks the purpose of the test. Something for another PR, I think.